base/float.jl

```diff
@@ -713,6 +713,8 @@ end
     eps(::Type{Float16}) = $(bitcast(Float16, 0x1400))
     eps(::Type{Float32}) = $(bitcast(Float32, 0x34000000))
     eps(::Type{Float64}) = $(bitcast(Float64, 0x3cb0000000000000))
+    eps(::Type{Complex{T}}) where {T<:AbstractFloat} = eps(T)
+    eps(x::Complex{T}) where {T<:AbstractFloat} = eps(norm(x))
```

Comment:
I would say to define `eps(z)` as follows:

* For any real number `x`, let `x̃ =fl(x)` be the closest floating-point value.  The key property of `ε=eps(x̃)` is that `|x - x̃| ≤ ε`.

* The corresponding thing should be true for complex numbers `z=a+ib` and the closest floating-point complex value `z̃=ã+ib̃=fl(a)+ifl(b)`:  we should have `|z - z̃| ≤ ε`.

This would suggest a definition `eps(z::Complex{T}) = hypot(eps(real(z)),eps(imag(z)))`.